.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+    # Manual dispatch for testing
+  workflow_dispatch:
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v2
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Publish package
+        run: ./gradlew packageWinX64 packageLinuxX64
```

Comment:
This doesn't need to run `./gradlew clean` first, right? Because each run is in a container?

Reply:
That's correct